packages/react/carbon.yml

```
@@ -0,0 +1,61 @@
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas/carbon-resources.schema.json
+library:
+  id: ibmdotcom-react
+  name: IBM.com React
+  description: IBM.com React components and patterns
+  noIndex: true
+  externalDocsUrl: www.ibm.com/standards/web/carbon-for-ibm-dotcom
+  inherits: ibmdotcom-styles
+  demoLinks:
+    - type: storybook
+      name: Storybook
+      action: link
+      url: https://www.ibm.com/standards/carbon/react/?path=/story/overview-getting-started--page
```

```suggestion
      url: https://www.ibm.com/standards/carbon/react
```

We can shorten this URL to ensure nothing breaks if they update their first page.